src/Elliptic/DiscontinuousGalerkin/InitializeFirstOrderOperator.hpp

```diff
@@ -71,7 +71,7 @@ struct InitializeFirstOrderOperator {
                                               vars_tag, PrimalVariables,
```

Comment:
The commit message is incorrect. The systems don't _need_ the fluxes to compute sources, but that having access to the fluxes is an optimization.